src/tools/rbd_ggate/main.cc

```diff
@@ -111,6 +111,7 @@ static int do_map(int argc, const char *argv[])
     global_init_postfork_start(g_ceph_context);
   }
 
+  g_ceph_context->_conf.finalize_reexpand_meta();
```

Comment:
Shouldn't it be in `if (global_init_prefork(g_ceph_context) >= 0)` block above? I suppose we don't need this if we are not forking.

Reply:
Yeah, this sounds a good idea.

Reply:
Maybe we should do this in `global_init_postfork_start(g_ceph_context);` instead.

Reply:
I am fine with this if it does not break anything.

Reply:
Checked it and didn't see it will break anything.
I have add @batrick and @tchaikov to help review this changes.